submodules/eks/node-group.tf

```diff
@@ -22,6 +22,10 @@ resource "aws_security_group" "eks_nodes" {
 
   lifecycle {
     create_before_destroy = true
+    ignore_changes = [
+      name,
```

Comment:
why are we ignoring name changes?  that seems very suspect.

Reply:
This is to cause less anxiety in the cdk->tf migration process. If we import the pre-existing node security group, then the old nodes can continue to function and Domino isn't broken at any phase of the process / you can validate that terraform didn't do anything destructive after the initial terraform run.

However, both the name and description fields are immutable and trigger a replacement:

```
  # module.domino_eks.module.eks.aws_security_group.eks_nodes must be replaced
+/- resource "aws_security_group" "eks_nodes" {
      ~ arn                    = "arn:aws:ec2:us-west-2:890728157128:security-group/sg-082c110d78d0a7170" -> (known after apply)
      ~ description            = "jhayes7916/EksStack/UnmanagedSG" -> "EKS cluster Nodes security group" # forces replacement
[...]
      ~ name                   = "jhayes7916-sharedNodeSG" -> "jhayes7916-nodes" # forces replacement
[...]
}
```

Since they are immutable, there should be no circumstance where this is triggered in a security group created by this module, only when importing into state as in the migration process. And since these fields are cosmetic, there should be no harm, either.